cli/functionary/config.py

```diff
@@ -52,7 +52,10 @@ def get_config_value(key):
         config_file = Path.home() / ".functionary" / "config"
         value = get_key(config_file, key)
         if value is None:
-            raise click.ClickException(f"Could not find value for {key}")
+            if raise_exception is not None:
```

Comment:
`raise_exception` is a boolean, so you actually want to be checking the value, or at least the truthiness of it.  As written, if someone passes in `True`, it won't actually raise an exception, because per this check, `True` is in fact, `not None`.

Also worth mentioning here, is that you should look at the other uses of `get_config_value` now that you've made this change with `raise_exception`.  Specifically, there's a use in `environment.py` that is has a try / except block but no longer needs it because this will now safely return `None`.